attack-theme/templates/resources/versions.html

```diff
@@ -0,0 +1,101 @@
+                            <td><a href="/resources/updates/{{parsed.current.changelog}}">Updates &mdash; {{parsed.current.changelog_label}}</a></td>
+                        </tr>
+                        {% for version in parsed.previous %}
+                        <tr>
+                            <td><a href="/versions/{{version.url}}">ATT&CK {{version.url}}</a></td>
+                            <td>{{version.date_start}}</td>
+                            <td>{{version.date_end}}</td>
+                            <td><a href="{{version.cti_url}}" target="_blank">{{version.name}} on MITRE/CTI</a></td>
+                            <td><a href="/resources/updates/{{version.changelog}}">Updates &mdash; {{ version.changelog_label }}</a></td>
+                        </tr>
+                        {% endfor %}
+                        <!-- versions that were on mediawiki preserved on the site -->
+                        <tr>
+                            <td colspan=5 class="table-break-row">
+                                Versions from before the migration from MediaWiki are not preserved on this site:
+                            </td>
+                        </tr>
+                        {% for version in parsed.older %}
+                        <tr>
+                            <td>ATT&CK {{version.url}}</td>
+                            <td>{{version.date_start}}</td>
+                            <td>{{version.date_end}}</td>
+                            <td><a href="{{version.cti_url}}" target="_blank">{{version.name}} on MITRE/CTI</a></td>
+                            <td><a href="/resources/updates/{{version.changelog}}">Updates &mdash; {{ version.changelog_label }}</a></td>
+                        </tr>
+                        {% endfor %}
+                    </tbody>
+                </table>
+
+                <!-- card display -->
+                <!-- <div class="card-deck">
+                    {% for version in parsed.versions %}
+                        <div class="card">
+                            <div class="card-header">
+                                <h5 class="mb-0">
+                                    <a href="/versions/{{version.name}}">ATT&CK {{version.name}}</a>
+                                </h5>
+                            </div>
+                            <div class="card-body">
+                                {{version.date_start}}&ndash;{{version.date_end}}
+                            </div>
+                            <div class="card-body">
+                                <a class="card-link" href="{{version.cti_url}}" target="_blank">data</a>
+                                <a class="card-link" href="/resources/updates/{{version.changelog}}" target="_blank">updates</a>
+                            </div>
+                        </div>
+                    {% endfor %}
+                </div> -->
```

Comment:
It appears to be dead code. If it is, we should remove it from the repository to reduce code base

Reply:
Yup you're right that's dead code. It was an alternative way of displaying the versions list. I'll remove it before I merge, thanks!